### PR TITLE
RHICOMPL-180 - Rule filtering by severity 

### PR DESCRIPTION
**Pull request by dLobatog**
Peek 2019-06-04 11-05

### CODE REVIEW

File: packages/inventory-compliance/src/RulesComplianceFilter.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+const FILTER_CATEGORIES = [
+    {
+        type: 'checkbox', title: 'Passed', urlParam: 'passed', values: [
+            { label: 'True', value: 'passed' },
+            { label: 'False', value: 'failed' }
+        ]
+    },
+    {
+        type: 'checkbox', title: 'Severity', urlParam: 'severity', values: [
```
Comment (katierik) [inline packages/inventory-compliance/src/RulesComplianceFilter.js]: Can we have the icons in this? It serves as a sort-of legend to have them
image

File: packages/inventory-compliance/src/RulesComplianceFilter.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { FilterDropdown } from '@redhat-cloud-services/frontend-components';
+
+const FILTER_CATEGORIES = [
+    {
+        type: 'checkbox', title: 'Passed', urlParam: 'passed', values: [
```
Comment (katierik) [inline packages/inventory-compliance/src/RulesComplianceFilter.js]: Updated mock for this. Uses radios and a little more description
image

**Comment by katierik** [comment]
Added 2 comments with mocks.

File: packages/inventory-compliance/src/RulesComplianceFilter.js (hunk 1 above)
Comment (dLobatog) [reply packages/inventory-compliance/src/RulesComplianceFilter.js]: Apparently the whole thing doesn't render if I try to display icons - https://github.com/RedHatInsights/frontend-components/pull/38 is meant to fix it

File: packages/inventory-compliance/src/RulesComplianceFilter.js (hunk 2 above)
Comment (dLobatog) [reply packages/inventory-compliance/src/RulesComplianceFilter.js]: Fixed

File: packages/inventory-compliance/src/RulesComplianceFilter.js (hunk 1 above)
Comment (dLobatog) [reply packages/inventory-compliance/src/RulesComplianceFilter.js]: Also that "Low" squared icon is nowhere to be found on https://patternfly-react.surge.sh/patternfly-4/icons/Icons/ :disappointed_relieved:

**Comment by katierik** [comment]
Re: different filter style: Yes there is a plan, but its not now (PF was going to deliver this, for now we're stuck with what we've got)

Re: low icon: Its a new one, we need it on the "severity" for CVEs too. Emailed to you.

File: packages/inventory-compliance/src/RulesComplianceFilter.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+class RulesComplianceFilter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            hidePassed: props.hidePassed,
+            severity: props.severity,
+            filterCategories: [
+                {
+                    type: 'radio', title: 'Passed', urlParam: 'hidePassed', values: [
+                        { label: 'Show all rules', value: false },
```
[the remark this answers is not in the document]
Comment (dLobatog) [reply packages/inventory-compliance/src/RulesComplianceFilter.js]: It already is :smile: 
Peek 2019-06-05 14-58

[the remark this answers is not in the document]
Comment (akofink) [reply packages/inventory-compliance/src/RulesComplianceFilter.js]: Yes, I confirm that this works as expected.

**Pull request merged**